lib/elixir/lib/map.ex

```diff
@@ -316,6 +316,9 @@ defmodule Map do
 
       iex> Map.fetch!(%{a: 1}, :a)
       1
+      iex> Map.fetch!(%{a: 1}, :b)
+      ** (KeyError) key :b not found in:
+      ...
```

Comment:
Given that it emits warnings today, perhaps it is best to convert it to a non-executable example indeed:

```suggestion

  When the key is missing, an exception is raised:
      
      Map.fetch!(%{a: 1}, :b)
      ** (KeyError) key :b not found in: %{a: 1}
```